src/ape/contracts/base.py

```diff
@@ -32,16 +31,11 @@ def __init__(
         self.abi = abi
         self.provider = provider
 
-    @validator("deployment_bytecode")
-    def check_empty_contract(cls, value):
-
+    def __post_init__(cls, value):
```

Comment:
I thought there was not a `__post_init__` though?

Can we just move this to inside the `__init__()`, (that was my suggestion before)

Reply:
Wait, what is `value`?

I am confused here

Reply:
I think this change may have been an accident?